phpBB/styles/prosilver/theme/common.css

```diff
@@ -1276,13 +1325,15 @@ ul.linklist:after,
 	padding: 8px;
 }
 
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	padding: 5px 0;
 }
 
 .dropdown-extended ul li a,
 .notification_list dt > a,
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	text-decoration: none;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12277453/25317994/d4666182-28a2-11e7-92a6-4ad180418e18.png)
In here as you can see pointer color is different. It is specific to dropdown-extended class. Moreover there are some styles which should be specific for smiley content. Such as width. So thought it's better to add a new class called smiley-content class and add those styles. It's easiar than using dropdown-extended class.